packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.2+4
+
+* Update version of json_annotation, resolve conflicts with other packages e.g. flutter_tools from sdk.
+
```

Comment:
maybe only:
`Update version of json_annotation to ^3.0.0 and json_serializable to ^3.2.0.` is enough?
cc @mklim

Reply:
I think either line is fine here. It's useful to know the exact version number and why we wanted to update it.

Reply:
Ok, minor edits on the CHANGELOG.md